### ISSUE TITLE
Read and implement "Sequence Classification with LSTM"

### ISSUE THREAD

**Issue by MadeInGer** [sagr4019/ResearchProject#52]
The goal is to compare our code with the code used in the article in order to find any issues why our code isn't running as expected.
The article is available at [machinelearningmastery.com](https://machinelearningmastery.com/sequence-classification-lstm-recurrent-neural-networks-python-keras/).

**Comment by jan-christiansen** [comment]
Execute example and list run times.

**Comment by jan-christiansen** [comment]
How does the data exactly look like.

**Comment by jan-christiansen** [comment]
What is the current state of this issue?

**Comment by MadeInGer** [comment]
The result of the first example:
![first_example_result](https://user-images.githubusercontent.com/30792417/41539363-eaa54cd6-730d-11e8-8f77-1e3ad323b678.PNG)

The result of the second example:
![second_example_result](https://user-images.githubusercontent.com/30792417/41539368-ed05d02c-730d-11e8-8d63-3023b741f6db.PNG)

The preprocessed dataset comes from https://s3.amazonaws.com/text-datasets/imdb.npz.
Original Dataset can be downloaded at http://ai.stanford.edu/~amaas/data/sentiment/.

A quote from the readme:

> The core dataset contains 50,000 reviews split evenly into 25k train
> and 25k test sets. The overall distribution of labels is balanced (25k
> pos and 25k neg). We also include an additional 50,000 unlabeled
> documents for unsupervised learning.

The text files are stored in different dirs (test/train, pos/neg)

A positive review looks like this:
![pos_example](https://user-images.githubusercontent.com/30792417/41539470-2503f224-730e-11e8-8389-6e1fa736912b.PNG)

A negative review looks like this:
![neg_example](https://user-images.githubusercontent.com/30792417/41539452-1aedf1ae-730e-11e8-861c-24d2ca6e8107.PNG)


An approach to preprocess the data is available at https://github.com/yoonkim/CNN_sentence. 
They used movie review data from Rotten Tomatoes, so they used a different data set.

Please read the short parahraph "Data and Preprocessing" at http://www.wildml.com/2015/12/implementing-a-cnn-for-text-classification-in-tensorflow/ to get additional information.

**Comment by jan-christiansen** [comment]
* What is "the first example" and what is "the second example"? Are these taken from the blog post? Please give a short explanation.
* Is "The preprocessed dataset" a preprocessed version of "Original dataset"? Give a short explanation, what "preprocessed" means.
* "A quote from the readme" Which readme?
* Was the approach for preprocessing data that you have linked to actually used to generate "The preprocessed dataset"?
* How is the paragraph "Data and Preprocessing" connect to this issue?
